helmfile/helmfile.yaml

```diff
@@ -0,0 +1,78 @@
+environments:
+  scratch:
+  dev:
+  staging:
+  production:
+---
+
+helmDefaults:
+  {{ if eq .Environment.Name "dev"}} 
+  kubeContext: dev
+  {{ end }}
+repositories:
+- name: secrets-store-csi-driver
+  url: https://kubernetes-sigs.github.io/secrets-store-csi-driver/charts
+- name: aws-secrets-manager 
+  url: https://aws.github.io/secrets-store-csi-driver-provider-aws
+- name: adfinis 
+  url: https://charts.adfinis.com
+- name: snowplow-devops 
+  url: https://snowplow-devops.github.io/helm-charts
+
+releases:
+
+  - name: ingress
+    namespace: nginx
+    labels:
+      app: nginx
+      tier: main
+      category: system
+    chart: oci://ghcr.io/nginxinc/charts/nginx-ingress
+    version: 1.1.2
+    values:
+    - ./overrides/system/nginx.yaml.gotmpl
+
+  - name: ingress-secrets
+    namespace: nginx
+    disableValidationOnInstall: true
+    labels:
+      app: nginx
+      tier: secrets
+      category: system
+    chart: adfinis/csi-secret-provider-class
```

Comment:
Let's have a `version` field for all referenced charts, for security purpose. 🙏

Reply:
This actually gets removed with latest changes since the chart is now local 🥳